source/renderer/zettlr-body.js

```diff
@@ -183,7 +188,8 @@ class ZettlrBody {
       return global.ipc.send('file-new', { 'name': generateFileName(), 'hash': dir.hash })
     }
 
-    let cnt = makeTemplate('popup', 'textfield', {
+    const textfieldTemplate = require('./../../resources/templates/popup/textfield.handlebars')
```

Comment:
For clarification: How is the handlebars runtime bundled with these? Is it so that webpack will pre-compile the `handlebars` during bundling, and this "require" actually requires the runtime with one of the templates? Or how does this work?

This is a strong indication that we need to comment this as well.